src/crypto/SignerKey.cpp

```diff
@@ -30,6 +30,8 @@ KeyFunctions<SignerKey>::getKeyVersionIsSupported(
         return true;
     case strKey::STRKEY_HASH_X:
         return true;
+    case strKey::STRKEY_ED25519_SIGNED_PAYLOAD:
+        return true;
```

Comment:
I'm not confident yet about the changes in this file. The `getKeyValue` methods below don't specify the `STRKEY_ED25519_SIGNED_PAYLOAD` case because I don't think it makes sense to return just the `ed25519` key for the payload signer. I'm pretty sure that the `getKeyValue` methods below are only called with `SIGNER_KEY_TYPE_ED25519` keys, but the change in this PR is error prone.

Reply:
Updated. The new `getKeyValue` method copies out the opaque content. I still think there's room for improvement though. These utils are pretty confusing.